cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -146,7 +147,7 @@ type Member struct {
 }
 
 // AddMember notifies an existing etcd cluster that a new member is joining
-func (c Client) AddMember(name string, peerAddrs string) ([]Member, error) {
+func (c *Client) AddMember(name string, peerAddrs string) ([]Member, error) {
```

Comment:
as a note/TODO: in a separate PR  we need to make all the client methods to use pointers.
golang encourages a matching pattern.